doc/changes/latest.rst

```diff
@@ -8,6 +8,8 @@
 Fixes
 -----
 
+- :bdg-dark:`Code` Fix issue to lead to user path to not be expanded when trying to load nifti images (:gh:`4948` by `Rémmi Gau`_).
```

Comment:
```suggestion
- :bdg-dark:`Code` Fix issue to lead to user path to not be expanded when trying to load nifti images (:gh:`4948` by `Rémi Gau`_).
```

Reply:
are you trying to tell I should take a break because I cannot spell my own name?

Reply:
It _has_ been a long week 🤣